src/entities/paginated_collection.coffee

```diff
@@ -137,7 +137,7 @@ define [], ->
     #
     # @param data [Object] the response returned from server
       parse: (data) ->
-        if Backbone.Collection.prototype.parse != collection.parse
+        if collection.parse? && (Backbone.Collection.prototype.parse != collection.parse)
```

Comment:
By default backbone collection has parse method defined on it . 

http://backbonejs.org/#Collection-parse

So there is no use of this collection.parse?